McCorrections/include/LeptonWeights.h

```diff
@@ -181,7 +181,7 @@ class LeptonWeights : public IWeightProvider {
             tauIdWeight = std::make_shared<TauIdWeight2016>();
         }
         else if(period == Period::Run2017){
-            tauTriggerWeight =  std::make_shared<TauTriggerWeight2017>(tauTriggerInput, tauTriggerInputOld, _tau_iso_wp);
+            tauTriggerWeight =  std::make_shared<TauTriggerWeight2017>(tauTriggerInput, tauTriggerInputOld, ToString(_tau_iso_wp));
```

Comment:
`TauTriggerSFs2017` expects that the wp string is all lower case. Use `boost::algorithm::to_lower_copy`.